test/test_turbulence_closures.jl

```diff
@@ -129,6 +130,36 @@ function time_step_with_tupled_closure(FT, arch)
     return true
 end
 
+function time_step_with_catke_closure(arch)
+    closure = CATKEVerticalDiffusivity()
+
+    model = NonhydrostaticModel(closure=closure,
+                                grid=RectilinearGrid(arch, size=(1, 1, 1), extent=(1, 2, 3)),
+                                buoyancy = BuoyancyTracer(),
+                                tracers = (:b, :e))
+
+    @test model.tracers.e.boundary_conditions.top.condition !== nothing
+
+    time_step!(model, 1, euler=true)
+
+    return true
+end
+
+function time_step_with_tupled_catke_closure(arch)
+    closure_tuple = (CATKEVerticalDiffusivity(), AnisotropicDiffusivity())
+
+    model = NonhydrostaticModel(closure=closure_tuple,
```

Comment:
Why not use `HydrostaticFreeSurfaceModel`? Isn't that our main use case?

Reply:
Good point.

But the test fail even with `NonhydrostaticModel`. In particular
```
@test model.tracers.e.boundary_conditions.top.condition !== nothing
```
fails! Seems like the top boundary condition for `e` is nothing?

Reply:
But I think we're ok if tests fail for `NonhydrostaticModel`, but _pass_ for `HydrostaticFreeSurfaceModel`.

Reply:
I'll try that!

Reply:
I'm not sure if that crucial function is in the constructor for `NonhydrostaticModel`. Do we support CATKE with `NonhydrostaticModel`?

Reply:
Looking into it, I don't think we support CATKE with `NonhydrostaticModel`.

Reply:
(but we should)

Reply:
until we do, should we print a warning?